scheduler/kubernetes/snaps.md

```diff
@@ -11,218 +11,111 @@ meta-description: "Learn to take a snapshot of a volume from a Kubernetes persis
 
-For using annotations Portworx daemon set requires extra permissions to read annotations from PVC object. Make sure your ClusterRole has the following section
+Once you apply the above object you can check the status of the snapshots using `kubectl`:
+
+```
+$ kubectl get volumesnapshot,volumesnapshotdatas 
+NAME                             AGE
+volumesnapshots/mysql-snapshot   2s
 
-```yaml
-- apiGroups: [""]
-  resources: ["persistentvolumeclaims"]
-  verbs: ["get", "list"]
+NAME                                                                            AGE
+volumesnapshotdatas/k8s-volume-snapshot-2bc36c2d-227f-11e8-a3d4-5a34ec89e61c    1s
 ```
 
-You can run the following command to edit your existing Portworx ClusterRole
+The creation of the volumesnapshotdatas object indicates that the snapshot has
+been created. If you describe the volumesnapshotdatas object you can see the
+Portworx Volume Snapshot ID and the PVC for which the snapshot was created.
 
 ```
-$ kubectl edit clusterrole node-get-put-list-role
+$ kubectl describe volumesnapshotdatas 
+Name:         k8s-volume-snapshot-2bc36c2d-227f-11e8-a3d4-5a34ec89e61c
+Namespace:    
+Labels:       <none>
+Annotations:  <none>
+API Version:  volumesnapshot.external-storage.k8s.io/v1
+Kind:         VolumeSnapshotData
+Metadata:
+  Cluster Name:                   
+  Creation Timestamp:             2018-03-08T03:17:02Z
+  Deletion Grace Period Seconds:  <nil>
+  Deletion Timestamp:             <nil>
+  Resource Version:               29989636
+  Self Link:                      /apis/volumesnapshot.external-storage.k8s.io/v1/k8s-volume-snapshot-2bc36c2d-227f-11e8-a3d4-5a34ec89e61c
+  UID:                            2bc3a203-227f-11e8-98cc-0214683e8447
+Spec:
+  Persistent Volume Ref:
+    Kind:  PersistentVolume
+    Name:  pvc-f782bf5c-20e7-11e8-931d-0214683e8447
+  Portworx Volume:
+    Snapshot Id:  991673881099191762
+  Volume Snapshot Ref:
+    Kind:  VolumeSnapshot
+    Name:  default/mysql-snapshot-2b2150dd-227f-11e8-98cc-0214683e8447
+Status:
+  Conditions:
+    Last Transition Time:  <nil>
+    Message:               
+    Reason:                
+    Status:                
+    Type:                  
+  Creation Timestamp:      <nil>
+Events:                    <none>
 ```
+### Creating PVCs from snapshots 
 
-##### Snapshot of a snapshot
+When you install STORK, it also creates a storage class called stork-snapshot-sc.
+This storage class can be used to create PVCs from snapshots.
 
-You can take a snapshot of a snapshot using the following spec file
+To create a PVC from a snapshot, you would add the
+`snapshot.alpha.kubernetes.io/snapshot` annotation to refer to the snapshot
+name.
 
-```yaml
-kind: PersistentVolumeClaim
+For the above snapshot, the spec would like this:
+```
 apiVersion: v1
+kind: PersistentVolumeClaim
 metadata:
-  namespace: prod
-  name: ns.prod-name.px-snap-2
+  name: mysql-snap-clone
```

Comment:
nit: we can call this `mysql-snap-restore` instead of clone.

Reply:
Restore has the connotation that it will restore to the original volume/PVC. Clone is the correct storage term to use.